great_expectations/render/view/view.py

```diff
@@ -434,3 +451,93 @@ def _validate_document(self, document):
         assert isinstance(
             document["content_block"], dict
         )  # For now low-level views take dicts
+
+
+class DefaultMarkdownPageView(DefaultJinjaView):
+    """
+    Convert a document to markdown format.
+    """
+
+    def _validate_document(self, document: RenderedDocumentContent) -> bool:
+        """
+        Validate that the document is of the appropriate type at runtime.
+        """
+        assert isinstance(document, RenderedDocumentContent)
+
+    _template = "markdown_page.j2"
+
+    def render_string_template(self, template: pTemplate) -> pTemplate:
+        """
+        Render string for markdown rendering. Bold all parameters and perform substitution.
+        Args:
+            template: python Template object
+
+        Returns:
+            Template with substituted values and all parameters bolded
+
+        """
+
+        if not isinstance(template, (dict, OrderedDict)):
+            return template
+
+        # if there are any groupings of two or more $, we need to double the groupings to account
+        # for template string substitution escaping
+        template["template"] = re.sub(
+            r"\${2,}", lambda m: m.group(0) * 2, template.get("template", "")
+        )
+
+        # Bold all parameters:
+        base_param_template_string = "**$content**"
+
+        # Make sure template["params"] is a dict
+        template["params"] = template.get("params", {})
+
+        if "markdown_status_icon" in template["params"]:
+            return template["params"]["markdown_status_icon"]
+
```

Comment:
This is fine for now, but can you add a TODO to this method to revisit how we handle rendering of icons in markdown? This works fine in this case since it's a discrete status_icon, but there could be other instances where there's an icon rendered inline with other text.